manifests/templates/reconciler-manager-configmap.yaml

```diff
@@ -111,7 +109,7 @@ data:
            imagePullPolicy: IfNotPresent
          - name: git-sync
            image: gcr.io/config-management-release/git-sync:v3.6.6-gke.3__linux_amd64
-           args: ["--root=/repo/source", "--dest=rev", "--max-sync-failures=30", "--error-file=error.json", "--v=5"]
+           args: ["--root=/repo/source", "--dest=rev", "--max-sync-failures=30", "--error-file=error.json"]
```

Comment:
do we not want git-sync to default to -v=5 any more? 
That was kinda useful for debugging, and not excessively verbose.
If so, we need different defaults for each container.

Reply:
Oh, I see this default was moved down to code in the reconciler-base. If it's still true, the defaults should probably be documented in the CRD field description.

Reply:
done, updated CRD with description of default value

Reply:
probably better to put the default values on the desc of the logLevel field itself, rather than a parent field.

Reply:
moved the desc to logLevel field